env0/resource_environment.go

```diff
@@ -81,11 +81,10 @@ func resourceEnvironment() *schema.Resource {
 				Default:     true,
 			},
 			"auto_deploy_by_custom_glob": {
-				Type:         schema.TypeString,
-				Description:  "redeploy on file filter pattern",
-				RequiredWith: []string{"auto_deploy_on_path_changes_only"},
-				Optional:     true,
-				Default:      "",
+				Type:        schema.TypeString,
+				Description: "redeploy on file filter pattern",
+				Optional:    true,
+				Default:     "",
```

Comment:
this could be redundant (I think* that d.get always returns the false value of that type)

Reply:
f99f7e4